app/src/app/poll/poll.component.sass

```diff
@@ -25,3 +25,7 @@
     width: 60px
   input
     text-align: center
+
+.poll-title, .poll-description, span.vote
+  hyphens: auto
+  -webkit-hyphens: auto
```

Comment:
This comes from another branch. New branches must start from main. See my comments on Slack for your future branch. For this PR, please merge origin/main into jw_scrollover.